O365/message.py

```diff
@@ -755,7 +756,20 @@ def reply(self, to_all=True):
             return None
 
         message = response.json()
+        message_body_content_html  = message['body']['content']
 
+        pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
+
+        def change_sent_time(match):
```

Comment:
Now that I understand the point of this PR, we should improve the naming of this function. We can name it `change_sent_time_timezone`, because technically we aren't changing the sent time itself, only the associated timezone

We should also add a quick comment about what this function does
```
"""Uses RegEx to extract the Sent time and attaches a timezone (self.protocol.timezone) to it"""
```